src/PropertyManager/Travely.PropertyManager.Service/Models/Base/FilteringBaseModel.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Travely.PropertyManager.Service.Models.Base
+{
+    public class FilteringBaseModel
+    {
+        public string FieldName { get; set; }
+        public string Value { get; set; }
+        public FilteringOperationType Type { get; set; }
+    }
+
+
+    public enum FilteringOperationType
```

Comment:
Please define in separate file and make enum start 1 to avoid default 0 value

Reply:
Fixed.